node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric_test.go

```diff
@@ -220,7 +228,7 @@ func TestGetMpathDevice(t *testing.T) {
 
 		{
 			name:             "Should fail when WaitForPathToExist return error with the first array iqn, and found no sd device with the second array iqn",
-			arrayIdentifiers: []string{"X", "Y"},
+			arrayIdentifiers: []string{"X", "X"},
```

Comment:
why are these necessary?

Reply:
X is identifier name. When error occures it contains X as part of error message
Before my change there was predefined order - first handle X, simulate result for for X, next same for Y.
Now it is not defined which identifier will be handled  so we simulate first